packages/components/src/composite/index.tsx

```diff
@@ -52,9 +52,44 @@ export const Composite = Object.assign(
 		HTMLDivElement,
 		WordPressComponentProps< CompositeProps, 'div', false >
 	>( function Composite(
-		{ children, store, disabled = false, ...props },
+		{
+			// Composite store props
+			activeId,
+			defaultActiveId,
+			setActiveId,
+			focusLoop = false,
+			focusWrap = false,
+			focusShift = false,
+			virtualFocus = false,
+			orientation = 'both',
+			rtl = false,
+
+			// Composite component props
+			children,
+			disabled = false,
+
+			// To be removed
+			store: storeProp,
+
+			// Rest props
+			...props
+		},
 		ref
 	) {
+		const newStore = Ariakit.useCompositeStore( {
+			activeId,
+			defaultActiveId,
+			setActiveId,
+			focusLoop,
+			focusWrap,
+			focusShift,
+			virtualFocus,
+			orientation,
+			rtl,
+		} );
+
+		const store = storeProp || newStore;
```

Comment:
Any good reason to prefer `||` to `??` here?

Reply:
Main reasons is that I copied the same style as the ariakit repo when dealing with `store` fallbacks:

https://github.com/ariakit/ariakit/blob/main/packages/ariakit-react-core/src/composite/composite.tsx#L145-L146

Also, `store` shouldn't have falsy values (`''`, `0`) which would cause `||` to behave differently than `??`